drivers/unix/net_socket_posix.cpp

```diff
@@ -194,7 +194,7 @@ NetSocketPosix::NetError NetSocketPosix::_get_socket_error() {
 		return ERR_NET_IS_CONNECTED;
 	if (errno == EINPROGRESS || errno == EALREADY)
 		return ERR_NET_IN_PROGRESS;
-	if (errno == EAGAIN || errno == EWOULDBLOCK)
+	if (errno == EWOULDBLOCK)
```

Comment:
As @akien-mga reported, some systems might have different values.
It seems that pretty much all modern OSes have the same value for both, but I would prefer to just suppress the warning here and avoid potential errors with exotics libc